src/pages/rama-issue/issue.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import "./issue.css";
+import { useNavigate } from 'react-router-dom';
+import Pic from "./Group2.jpg";
```

Comment:
It is better to use svg as it conserves space and therefore takes less time to load, also the picture will not be pixelated